components/public-api/gitpod/v1/user.proto

```diff
@@ -81,6 +81,7 @@ message UpdateUserRequest {
     optional string signup_goals_other = 10;
     optional string onboarded_timestamp = 11;
     optional string company_size = 12;
+    map<string, string> coachmarks_dismissals = 13;
```

Comment:
I hope to use this for future dialogs of this nature to not bloat the profile object.